src/main/java/org/broadinstitute/hellbender/engine/spark/SparkContextFactory.java

```diff
@@ -153,4 +156,36 @@ private static JavaSparkContext createTestSparkContext(Map<String, String> overr
         final SparkConf sparkConf = setupSparkConf("TestContext", DEFAULT_SPARK_MASTER, DEFAULT_TEST_PROPERTIES, overridingProperties);
         return new JavaSparkContext(sparkConf);
     }
+
+    /**
+     * Determine the number of cores Spark master should use. Applicable to Spark test only.
```

Comment:
Reword method description slightly: "Creates the default Spark master, and determines the number of cores it should use."